### PR TITLE
[RFR] Rewrite sustainable open source page

### PR DESCRIPTION
**Pull request by SimonBiggs**

### CODE REVIEW

File: docs/supporters/sustainable.rst [hunk 1]
```diff
@@ -2,22 +2,60 @@
...
 =======================
 
-..    include:: <isonum.txt>
+Project Maintenance
+-------------------
 
-PyMedPhys is supported by companies through employee developer time as well as
-the purchasing of advertising space. This is a fundamental part of PyMedPhys
-having an open and vibrant future.
+In its current form, at the time of writing, there are five contributors, two
```
Comment (Centrus007) [inline docs/supporters/sustainable.rst]:
```suggestion
In its current form, at the time of writing, there are five contributors. Two
```

**Comment by Centrus007** [comment]
See my latest commit for suggested edits :)

**Comment by Centrus007** [comment]
@SimonBiggs I think it's best if you review my suggestions and merge rather than me going ahead. This is an important page, and some of my suggestions constituted reasonable rewords...

File: docs/supporters/sustainable.rst [hunk 2]
```diff
@@ -1,12 +1,12 @@
-=======================
+#######################
 Sustainable Open Source
-=======================
+#######################
```
Comment (SimonBiggs) [inline docs/supporters/sustainable.rst]: This is something I don't yet understand with reST, what's the difference here?

File: docs/supporters/sustainable.rst [hunk 3]
```diff
@@ -1,23 +1,73 @@
...
+
+We would like to implement minimally-intrusive advertising. We will provide a
+user-friendly and integrated way for companies who want to become supporters to
+bid for ad placement.
+
+All advertising will be based upon `viewable impressions
+<https://www.thinkwithgoogle.com/feature/viewability/#/demo>`_
+using `a transparent open source viewability library
+<https://kahwee.github.io/viewability/timer.html>`_. We would specifically
+exclude charging for advertising to any user who is a maintainer of the
```
Comment (SimonBiggs) [inline docs/supporters/sustainable.rst]:
```suggestion
exclude charging for advertising when the advert is viewed by a maintainer of the
```

**Comment by SimonBiggs** [comment]
Yup, I'm happy on that reworking. I've added one commit suggestion. Once you're happy with that feel free to merge.

File: docs/supporters/sustainable.rst (hunk 2 above)
Comment (Centrus007) [reply docs/supporters/sustainable.rst]: See the [Sphinx guide for Sphinx-based documentations](https://documentation-style-guide-sphinx.readthedocs.io/en/latest/style-guide.html#headings).

'###' is meant for doc titles, '===' denotes a H3 subheading. I didn't see a formatting change though, so I think Sphinx might be promoting the highest level subheading to title and scaling every subheading accordingly...

**Pull request merged**